### DISCUSSION TITLE
Plan for 2.2?

### DISCUSSION THREAD

**Discussion by tesseract-two** [drehmal/wiki#15] [Q&A] [answered]
Are you planning to keep around the old 2.1 documentation or deleting it and replacing it with 2.2 documentation?

**Comment by Zottelchen** [comment, marked as answer]
Similiar to a real wiki new content will replace old content. You either have the whole history in this Github Repo or I can give you links to certain older versions (e.g. https://f4b900fa.drehmal-wiki.pages.dev/)
